variables.tf

```diff
@@ -518,6 +530,12 @@ variable "sns_kms_master_key_id" {
   default     = "alias/aws/sns"
```

Comment:
Is this value for a kms key?

Reply:
This is the alias of the AWS managed AWS KMS for Amazon SNS